src/content/docs/dropins/payment-services/containers/apple-pay.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: ApplePay container
```

Comment:
we should use one only everywhere, except when it's code: either Apple Pay, or ApplePay